src/allotropy/parsers/bmg_mars/bmg_mars_parser.py

```diff
@@ -133,15 +134,13 @@ def _get_read_type(self, lines: list[str]) -> ReadType:
             re.search(RE_READ_TYPE, "\n".join(lines), flags=re.MULTILINE),
             msg="Read type not found.",
         ).group(0)
-        patterns = {
-            "Absorbance": ReadType.ABSORBANCE,
-            "Fluorescence": ReadType.FLUORESCENCE,
-        }
 
-        for key in patterns:
-            if key in read_type:
-                read_type_key = patterns[key]
-        return read_type_key
+        if "Absorbance" in read_type:
```

Comment:
Just confirming that if these are somehow both in the read_type that "Absorbance" takes precedence? Sorry this may just be a nonsensical scenario

Reply:
Updated to explicitly exclude the scenario of >1 read_type.